checkoutmanager/dirinfo.py

```diff
@@ -156,6 +185,24 @@ class BzrDirInfo(DirInfo):
     vcs = 'bzr'
 
     @capture_stdout
+    def cmd_rev(self):
+        print(self.directory)
+        os.chdir(self.directory)
+        print(system("bzr log -r-1"))
+
+    @capture_stdout
+    def cmd_cu(self):
+        output = system("bzr missing")
+        output = output.strip()
+        output_lines = output.split('\n')
+        if "Branches are up to date." in output_lines:
```

Comment:
I agree that this new command is separate from the existing ones. So something that's mostly working is fine and we can tweak it later when needed.

So what you have now is fine (apart from adding a `.lower()`), though you are free to experiment with that "bzr log -r -1" stuff of course :-)